pallets/ddc-payouts/src/lib.rs

```diff
@@ -270,6 +271,8 @@ pub mod pallet {
 	pub struct BillingReport<T: Config> {
 		pub state: State,
 		pub vault: T::AccountId,
+		pub start_era: i64,
+		pub end_era: i64,
```

Comment:
`BillingReport` is stored inside a storage called `ActiveBillingReport` which means that adding additional fields would require a storage migrations. 

If you want to skip creating a migrations, you have to make sure that all `ActiveBillingReport` storages in all test chains & mainnet are empty.

Reply:
the only component that uses Payout pallet is DAC and it hasn't been deployed anywhere so there are no existing billing reports and the storage is empty -> no migration required